samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -626,3 +626,15 @@ seb_download_seb=Download Safe Exam Browser
 seb_launch_seb=Launch Safe Exam Browser
 seb_quit_browser=Quit Safe Exam Browser
 seb_preview_info=Assessments that are configured to use Safe Exam Browser can not be previewed.
+
+#S2U-16 Timed Part/Question
```

Comment:
Hey @PabloSanRoman, I just found more missing properties for S2U-16. Could you translate them as well please?
```
timed=Timed

timer_start=Start
partTimer_title=Time remaining for this part
partTimer_info_start=Timed part. Once you click \"Start\" you will have <b>{0}</b> to complete this part. It will be <b>submitted at that time</b>, regardless of whether you have answered all the questions.
partTimer_info_start_deadline=Timed part. Once you click \"Start\" you will have <b>{0} or until {1} whichever is shorter</b> to complete this part. It will be <b>submitted at that time</b>, regardless of whether you have answered all the questions.
partTimer_info_end=The time allowed for this part has expired.
questionTimer_title=Time remaining for this question
questionTimer_info_start=Timed question. Once you click \"Start\" you will have <b>{0}</b> to complete this question. It will be <b>submitted at that time</b>, regardless of whether you have answered it.
questionTimer_info_start_deadline=Timed question. Once you click \"Start\" you will have <b>{0} or until {1} whichever is shorter</b> to complete this question. It will be <b>submitted at that time</b>, regardless of whether you have answered it.
questionTimer_info_end=The time allowed for this question has expired.
title_not_available=Title not available yet

part_timed_info=If timed, students have until the end of the time limit, or until the Final Submission Deadline, whichever comes first, to submit their work. Their answers will be automatically saved and submitted at the expiration of the timer.
question_timed_info=If timed, students have until the end of the time limit, or until the Final Submission Deadline, whichever comes first, to submit their work. Their answers will be automatically saved and submitted at the expiration of the timer.
part_timeSelect_error=Please specify a time limit in hours and/or minutes, or select the "No" option.
question_timeSelect_error=Please specify a time limit in hours and/or minutes, or select the "No" option.
```

Reply:
Hi @stetsche , here they are: 
timed=Tenporizatua

timer_start=Hasi
partTimer_title=Falta den denbora parte honetan
partTimer_info_start=Parte tenporizatua. "Hasi" botoia sakatu ondoren {0} izango duzu parte hau bukatzeko. Une horretan bidaliko da, galdera guztiak erantzun dituzun edo ez alde batera utzita.
partTimer_info_start_deadline= Parte tenporizatua. "Hasi" botoia sakatu ondoren <b>{0} izango duzu partea bukatzeko, edo data hau arte, {1}, (lehenengo gertatzen dena)</b>. Une horretan <b>bidaliko da</b>, galdera guztiak erantzun dituzun edo ez alde batera utzita.
partTimer_info_end=Amaitu da parte hau egiteko denbora.
questionTimer_title=Galdera honetarako geratzen den denbora
questionTimer_info_start=Galdera tenporizatua. "Hasi" botoia sakatu ondoren, {0} izango duzu galderari erantzuteko. Une horretan bidaliko da, erantzun duzun edo ez alde batera utzita. 
questionTimer_info_start_deadline=Galdea tenporizatua. "Hasi" botoia sakatu ondoren <b>{0} izango duzu galderari erantzuteko, edo data hau arte, {1}, (lehenengo gertatzen dena)</b>. Une horretan <b>bidaliko da</b>, erantzun duzun edo ez alde batera utzita.
questionTimer_info_end=Galdera hau egiteko denbora agortu da. 
title_not_available=Titulua ez dago erabilgarria oraindik

part_timed_info=Parte hau tenporizatuta badago ikasleek azterketa bidali ahalko dute denbora-muga hori amaitu arte edo azterketaren "Entregatzeko data berantiarra" arte, lehenengo gertatzen dena. Erantzunak automatikoki gorde eta bidaliko dira tenporizadorea amaitzen denean.
question_timed_info=Tenporizatuta badago, ikasleek azterketa bidali ahalko dute denbora-muga hori amaitu arte edo azterketaren "Entregatzeko data berantiarra" arte, lehenengo gertatzen dena. Erantzunak automatikoki gorde eta bidaliko dira tenporizadorea amaitzen denean.
part_timeSelect_error=Zehaztu denbora-muga bat ordu eta/edo minututan, edo hautatu "Ez" aukera.

Reply:
Thanks @PabloSanRoman !